misc/services/mobile-webui/mobile-webui-frontend/src/apps/huManager/api.js

```diff
@@ -36,3 +36,25 @@ export const moveHU = ({ huId, huQRCode, targetQRCode }) => {
     .then(unboxAxiosResponse)
     .then((response) => response.result);
 };
+
+/**
+ * @returns {Promise<T>} handling unit info
+ */
+export function getAllowedClearanceStatusesRequest({ huId }) {
+  return axios
+    .get(`${huAPIBasePath}/byId/${huId}/allowedClearanceStatuses`)
+    .then(unboxAxiosResponse)
+    .then((response) => response.clearanceStatuses);
+}
+
+export function setClearanceStatusRequest({ huId, clearanceNote, clearanceStatus }) {
+  const dataObject = {};
+  if (clearanceNote) {
+    dataObject.clearanceNote = clearanceNote;
+  }
+  if (clearanceStatus) {
```

Comment:
what's the reason for not having a clearanceStatus?

Reply:
> If it didn't change or user only added clearanceNote, should we send it anyway ?

Nop, a clearanceNote makes no sense without a status.
The `note` is the explanation of the `status`.

e.g. Status:Quarantine, Note: A bird took a dump on it